### DISCUSSION TITLE
1.2.3

### DISCUSSION THREAD

**Discussion by steccas** [Steccas/ProtonClient#11] [General]
# Changelog
- security patches
- updated dependencies

If you prefer there are also some build archives, which you can unpack and run where you prefer.

PLEASE NOTE:
If you want to use the client you need to create a protonMail account [here](https://go.getproton.me/SHJN).

# Install Instructions
## Windows
Uninstall previous version and install via the provided msi.

## Ubuntu & Debian based Linux distros
Better remove previous version for cleaner install
```
sudo apt remove proton-client
```
Install the deb package using
```
sudo apt install ./proton-client-1.2.3-amd64.deb
```
or
```
sudo dpkg -i ./proton-client-1.2.3-amd64.deb
```
## RHEL based distros
RPM is not tested because I converted it using alien, so please notify me if there are any problems.

Install using:
```
sudo rpm –i proton-client-1.2.3-2.x86_64.rpm
```

To uninstall:
```
sudo rpm –e proton-client-1.2.3-2.x86_64.rpm
```

## Arch based distros like Manjaro
Download **proton-client-1.2.3-1-x86_64.arch.zip**
```
unzip proton-client-1.2.3-1-x86_64.arch.zip
sudo pacman -U ./proton-client-1.2.3-1-x86_64.pkg.tar.zst
```

I have no time nor patience to publish the package to AUR so any help is welcome.

## Mac
Use one of the archives for Mac (proton-client-1.2.3-mac-x86_64.zip); inside, you will find an app bundle ready to be moved when makes sense to you (probably the app folder) and being run.

<hr /><em>This discussion was created from the release <a href='https://github.com/Steccas/ProtonClient/releases/tag/'>1.2.3</a>.</em>

**Comment by mecrally** [comment]
Hi sir,
sorry I have downloaded the app from the provided .msi (I have Win 11 Pro 25 H2) but I see:

<img width="558" height="174" alt="1" src="https://github.com/user-attachments/assets/060de3f0-3465-45cc-8dac-783db3ae9a43" />

<img width="1426" height="893" alt="2" src="https://github.com/user-attachments/assets/c73841f4-8059-4fc5-bdf3-9213259e2d80" />

I don't want to pay for Proton Bridge... it's quite expensive for my need
Could you help me?

PS
I have donated right now
<img width="1268" height="628" alt="image" src="https://github.com/user-attachments/assets/78908bce-cf48-40a5-b0b9-559c57071cc7" />

**Comment by mecrally** [comment]
Hi @steccas thanks for your update. Still I can not use it. Tried with Mac

<img width="1346" height="824" alt="Screenshot 2026-02-23 at 01 30 44 AM" src="https://github.com/user-attachments/assets/b72cb773-b4cf-4117-b961-e483adce0ea0" />


Dave
